src/file-size/format.js

```diff
@@ -0,0 +1,20 @@
+import FileSizeFormatter from './format-file-size';
+import localeProvider from '../locale-provider';
+
+export default class FileSizeFormat {
+	constructor(locale, options) {
+		this.options = options || {};
+		this.localeData = localeProvider(locale, options.locale);
+	}
+	formatFileSize(fileSize, options){
+		options = options || {};
```

Comment:
I don't think we need `options` here (yet). Otherwise people will override the number of decimals _because they can_ and we'll end up with inconsistencies across UIs. Better to just pick a good default (same as what we do in the LMS) and only add an override later if it's really needed.